src/intersection-observer.js

```diff
@@ -218,10 +223,18 @@ export class IntersectionObserver extends Observable {
   }
 
   /**
-   * Provice a function to clear timeout before set this intersection to null.
+   * Provide a function to clear timeout before set this intersection to null.
    */
   destroy() {
     timer.cancel(this.flushTimeout_);
```

Comment:
I'd still go ahead and call `onViewportCallback(false)` in destroy for double tap (https://www.youtube.com/watch?v=HQe2f8L-Rkc)